File: Practica-cpp/Trabajo_practicos/4-Ciclos_o_Bucles/Ejercicio_3.cpp
/*
 * Realice un programam que calcule y muestre en la salida estandar la suma de los cuadrados de los 10 primeros enteros mayores que cero
 */

#include <iostream>
using namespace std;
int main() {
    int suma = 0,cuadrado;

    for (int i = 1; i <= 10; i++) {
        cuadrado = i * i;
        suma+=cuadrado;

    }
    cout << "EL resultado de la suma es:"<<suma << endl;
    cin.get();
    return 0;
}

File: Practica-java/src/_4_Ciclos_O_Bucles/Ejercicio_3.java
package _4_Ciclos_O_Bucles;
import java.util.Scanner;
/*
 * Realice un programam que calcule y muestre en la salida estandar la suma de los cuadrados de los 10 primeros enteros mayores que cero
 */
public class Ejercicio_3 {
    public static void main(String[] arg){
        Scanner entrada = new Scanner(System.in);
        int suma = 0,cuadrado;

        for (int i = 1; i <= 10; i++) {
            cuadrado = i * i;
            suma+=cuadrado;

        }
        System.out.println("EL resultado de la suma es:"+suma);
    }
}
